rocAL/rocAL/include/context.h

```diff
@@ -53,6 +53,8 @@ struct Context
     }
     size_t user_batch_size() { return _user_batch_size; }
     size_t internal_batch_size() { return _internal_batch_size; }
+    void set_user_batch_size(size_t user_batch_size) {  _user_batch_size = user_batch_size; }
+    void set_internal_batch_size(size_t internal_batch_size) { _internal_batch_size = internal_batch_size; }
```

Comment:
In the previous discussion, I thought we have discussed to not to set internal_batch_size externally